admin_guide/scheduling/node_affinity.adoc

```diff
@@ -262,12 +269,14 @@ The following example demonstrates node affinity for a node and pod without matc
 
```

Comment:
Line 260-262 can be updated with the new `[source,terminal]` syntax. I know this PR isn't comprehensive, but since that's in the same article as others that were updated, it probably makes sense to update it too.